backend/app/views/spree/admin/promotion_code_batches/index.html.erb

```diff
@@ -0,0 +1,65 @@
+<% admin_breadcrumb(link_to plural_resource_name(Spree::Promotion), spree.admin_promotions_path) %>
+<% admin_breadcrumb(link_to @promotion.name, spree.edit_admin_promotion_path(@promotion.id)) %>
+<% admin_breadcrumb(plural_resource_name(Spree::PromotionCodeBatch)) %>
+
+<% content_for :page_actions do %>
+  <li>
+    <% if can?(:create, Spree::PromotionCodeBatch) %>
+      <%= button_link_to Spree.t(:new_promotion_code_batch), new_object_url %>
+    <% end %>
+  </li>
+<% end %>
+
+<% if @promotion_code_batches.any? %>
+  <table>
+    <thead>
+      <tr>
+        <th><%= Spree.t("promotion_code_batches.base_code") %></th>
```

Comment:
Please don't invent your own translation keys, if Rails already has one for you.
Use `Spree::PromotionCodeBatch.human_attribute_name(:COLUMN_NAME)` instead.

Thanks